pkg/util/startuplogger/startuplogger.go

```diff
@@ -0,0 +1,81 @@
+package startuplogger
```

Comment:
I think we could place this package in `pkg/cmd/servicemanager/SUBDIR`.
I don't think this should be util - it's used in (almost) single place, for single purpose.

Reply:
moved